packages/salesforcedx-vscode-core/src/commands/util/betaDeployRetrieve.ts

```diff
@@ -4,37 +4,23 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-
-import { ComponentSet, MetadataType } from '@salesforce/source-deploy-retrieve';
+import { getRelativeProjectPath } from '@salesforce/salesforcedx-utils-vscode/out/src';
 import { MetadataComponent } from '@salesforce/source-deploy-retrieve';
-import * as vscode from 'vscode';
-import { sfdxCoreSettings } from '../../settings';
-
-export function useBetaDeployRetrieve(
```

Comment:
This function isn't really needed anymore - it was from when we needed to detect the type before deploying or retrieving in addition to checking the setting. Now we can just check the setting. I got rid of it now because it was actually causing problems - If we couldn't infer the type here no deploy would trigger at all.